### PR TITLE
History tags: restyle and reword

### PR DESCRIPTION
**Pull request by stephanfowler**
**Mega nav**

screen shot 2015-01-21 at 14 25 35

**Preferences page**

screen shot 2015-01-21 at 14 17 51

Displays 10 tags max.

### CODE REVIEW

File: static/src/stylesheets/module/nav/_navigation.scss [hunk 1]
```diff
@@ -406,6 +412,26 @@ $c-navigation-expandable-background: colour(neutral-1);
...
+            margin-left: $gs-gutter / 4;
+            margin-right: 0;
+            margin-bottom: 0;
+
+          &:hover {
+             text-decoration: none;
+          }
+        }
+        @include mq($until: mobileLandscape) {
+            & {
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/nav/_navigation.scss]: I might be being dumb here. Why do you need this selector? Wouldn't this do the job...

```sass
@include mq($until: mobileLandscape) {
    margin-left: 0;
}
```

Comment (stephanfowler) [inline static/src/stylesheets/module/nav/_navigation.scss]: Err... yes. Dunno how that got there.

Comment (sammorrisdesign) [inline static/src/stylesheets/module/nav/_navigation.scss]: I'd probably also move it above `.button--tag` so it's closer to the the actual selector. Maybe.

**Comment by phamann** [comment]
:+1: Shipton

**Pull request merged**